content/getting-started/networkSettings/_index.en.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Network Settings"
+date: 2022-02-22T16:03:11.470Z
+lastmod: 2022-02-22T16:03:11.470Z
+weight: 2
+draft: false
+# search related keywords
+keywords: ["mac, macos, windows, linux, ios, android"]
+---
+
```

Comment:
Let's have an Info notice here - so new beginners are not mistaken about what this is and whether it's required.

Please do NOT change to testnet unless you know what you are doing. "The testnet is an alternative Bitcoin blockchain to be used for testing. Testnet coins are separate and distinct from actual bitcoin, and are never supposed to have any value. This allows application developers or bitcoin testers to experiment, without having to use real bitcoin or worrying about breaking the main bitcoin chain." (From https://en.bitcoin.it).